MetovaTestKitTests/Testable Protocol Tests/TestableProtocolTests.swift

```diff
@@ -59,4 +59,15 @@ class TestableProtocolTests: MTKBaseTestCase {
         
         waitForExpectations(timeout: 0, handler: nil)
     }
+    
+    func testErrorIsThrown() {
```

Comment:
It maybe seems backwards/redundant/odd, but I think pedantically, we should also assert that no error is thrown in the case when the `test` block itself doesn't throw an error.